.github/workflows/_mac-build.yml

```diff
@@ -71,12 +66,9 @@ jobs:
           fi
 
       - name: Setup miniconda
-        uses: conda-incubator/setup-miniconda@v2
+        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
```

Comment:
Nitpicking: should we use a particular tag rather than main? To prevent changes to test-infra accidentally regression CI?

Reply:
We'd have to tag test-infra in order to be able to do that, which I don't think we're really ready to do right now